packages-web/maps/src/utils/index.ts

```diff
@@ -0,0 +1,9 @@
+export declare type Option<T> = T | undefined;
+export * from "./data";
```

Comment:
Do you need this re-exports? It is not very hard to import from the actual file

Reply:
Done